.github/workflows/hivemq-operator-integration-test.yml

```diff
@@ -19,7 +19,8 @@ jobs:
       matrix:
         test-plan: [ K8sVersionCompatibility, Extensions, RollingUpgrades, Other ]
     steps:
-      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
+      - name: Checkout HiveMQ Platform Helm Charts
```

Comment:
Not sure about the name `HiveMQ Platform Helm Charts` - it may lead to think we are actually checking out only the Platform Helm charts. I would may be just call it `Checkout HiveMQ Helm Charts` - WDYT?

Reply:
Then we need to align this in all actions, I've copied this from an existing checkout :sweat_smile: But yeah, I will do the rename everywhere.